src/components/lib/pci/doc.md

```diff
@@ -0,0 +1,9 @@
+## pci
```

Comment:
Same here. I think that you can add a brief description, and state that an assumption is that we're only using, what, pci 1.0 here? You can add that a todo is to add support for pcie etc...